apps/samples/Ensemble8/src/samples/java/ensemble/samples/language/swing/SwingInterop.java

```diff
@@ -218,8 +218,8 @@ public void run() {
     }
 
     private Pane createBrowser() {
-        Double widthDouble = new Integer(PANEL_WIDTH).doubleValue();
-        Double heightDouble = new Integer(PANEL_HEIGHT).doubleValue();
+        Double widthDouble = Integer.valueOf(PANEL_WIDTH).doubleValue();
+        Double heightDouble = Integer.valueOf(PANEL_HEIGHT).doubleValue();
```

Comment:
I think `Double.valueof(PANEL_WIDTH)` would be simpler? Or, you can use auto-boxing and just say:

```
    Double widthDouble = PANEL_WIDTH;
    Double heightDouble = PANEL_HEIGHT;
```